config.yaml

```diff
@@ -26,6 +26,98 @@ options:
     default: false
     type: boolean
     description: Enable unaccent extension
+  plugin_bloom_enable:
+    default: false
+    type: boolean
+    description: Enable bloom extension
+  plugin_btree_gin_enable:
+    default: false
+    type: boolean
+    description: Enable btree_gin extension
+  plugin_btree_gist_enable:
+    default: false
+    type: boolean
+    description: Enable btree_gist extension
+  plugin_cube_enable:
+    default: false
+    type: boolean
+    description: Enable cube extension
+  plugin_dict_int_enable:
+    default: false
+    type: boolean
+    description: Enable dict_int extension
+  plugin_dict_xsyn_enable:
+    default: false
+    type: boolean
+    description: Enable dict_xsyn extension
+  plugin_earthdistance_enable:
+    default: false
+    type: boolean
+    description: Enable earthdistance extension
+  plugin_fuzzystrmatch_enable:
+    default: false
+    type: boolean
+    description: Enable fuzzystrmatch extension
+  plugin_intarray_enable:
+    default: false
+    type: boolean
+    description: Enable intarray extension
+  plugin_isn_enable:
+    default: false
+    type: boolean
+    description: Enable isn extension
+  plugin_lo_enable:
+    default: false
+    type: boolean
+    description: Enable lo extension
+  plugin_ltree_enable:
+    default: false
+    type: boolean
+    description: Enable ltree extension
+  plugin_old_snapshot_enable:
+    default: false
+    type: boolean
+    description: Enable old_snapshot extension
+  plugin_pg_freespacemap_enable:
+    default: false
+    type: boolean
+    description: Enable pg_freespacemap extension
+  plugin_pgrowlocks_enable:
+    default: false
+    type: boolean
+    description: Enable pgrowlocks extension
+  plugin_pgstattuple_enable:
+    default: false
+    type: boolean
+    description: Enable pgstattuple extension
+  plugin_pg_visibility_enable:
+    default: false
+    type: boolean
+    description: Enable pg_visibility extension
+  plugin_seg_enable:
+    default: false
+    type: boolean
+    description: Enable seg extension
+  plugin_tablefunc_enable:
+    default: false
+    type: boolean
+    description: Enable tablefunc extension
+  plugin_tcn_enable:
+    default: false
+    type: boolean
+    description: Enable tcn extension
+  plugin_tsm_system_rows_enable:
+    default: false
+    type: boolean
+    description: Enable tsm_system_rows extension
+  plugin_tsm_system_time_enable:
+    default: false
+    type: boolean
+    description: Enable tsm_system_time extension
+  plugin_uuid-ossp_enable:
```

Comment:
```suggestion
  plugin_uuid_ossp_enable:
```

Reply:
Unfortunately, the `-` symbol is prohibited for config options, please implement a workaround for extensions with `-` in the name.